cdc/model/sink.go

```diff
@@ -183,10 +185,10 @@ func (b *ColumnFlagType) UnsetIsUnsigned() {
 
 // TableName represents name of a table, includes table name and schema name.
 type TableName struct {
-	Schema      string `toml:"db-name" json:"db-name"`
-	Table       string `toml:"tbl-name" json:"tbl-name"`
-	TableID     int64  `toml:"tbl-id" json:"tbl-id"`
-	IsPartition bool   `toml:"is-partition" json:"is-partition"`
+	Schema      string `toml:"db-name" json:"db-name" msg:"db-name"`
+	Table       string `toml:"tbl-name" json:"tbl-name" msg:"tbl-name"`
+	TableID     int64  `toml:"tbl-id" json:"tbl-id" msg:"tbl-id"`
+	IsPartition bool   `toml:"is-partition" json:"is-partition" msg:"is-partition"`
```

Comment:
```suggestion
	Schema      string `toml:"db-name" json:"db-name" msg:"dbName"`
	Table       string `toml:"tbl-name" json:"tbl-name" msg:"tableName"`
	TableID     int64  `toml:"tbl-id" json:"tbl-id" msg:"tableID"`
	IsPartition bool   `toml:"is-partition" json:"is-partition" msg:"isPartition"`
```

For msgpack encoding, it's better to unify naming style.

Reply:
It looks like the official separator is using _. https://github.com/tinylib/msgp/blob/43b487371e0cdd1a3fe1be9a7802d8bce917a1af/tinygotest/testdata/roundtrip/main.go# L33